optimum/exporters/openvino/convert.py

```diff
@@ -374,13 +371,12 @@ def ts_patched_forward(*args, **kwargs):
 
                     __make_16bit_traceable(model)
                 check_dummy_inputs_are_allowed(model, dummy_inputs)
-                sig = inspect.signature(model.forward) if hasattr(model, "forward") else inspect.signature(model.call)
-                inputs = config.ordered_inputs(model)
-                input_names = list(inputs.keys())
-                output_names = list(config.outputs.keys())
-                input_info = get_input_shapes(dummy_inputs, inputs)
-
-                ov_model = convert_model(model, example_input=dummy_inputs, input=input_info)
+                input_info = get_input_info(model, config, dummy_inputs)
+                ov_model = convert_model(
+                    model,
+                    example_input=dummy_inputs,
+                    input=list(map(lambda item: (item.shape, item.type), input_info)),
```

Comment:
```suggestion
                    input=[(item.shape, item.type) for item in input_info],
```

will also fix the quality test https://github.com/huggingface/optimum-intel/actions/runs/10789831892/job/29923669303?pr=696

```
optimum/exporters/openvino/convert.py:378:27: C417 Unnecessary `map` usage (rewrite using a `list` comprehension)
optimum/exporters/openvino/convert.py:417:23: C417 Unnecessary `map` usage (rewrite using a `list` comprehension)
optimum/exporters/openvino/utils.py:89:18: C408 Unnecessary `list` call (rewrite as a literal)
optimum/exporters/openvino/utils.py:91:22: C408 Unnecessary `dict` call (rewrite as a literal)
```